genie-agent/src/test/java/com/netflix/genie/agent/cli/package-info.java

```diff
@@ -0,0 +1,25 @@
+/*
+ *
+ *  Copyright 2018 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+
+/**
+ * Commands for Agent and their command-line arguments descriptors.
+ *
+ * @author mprimi
+ * @since 4.0.0
+ */
+package com.netflix.genie.agent.cli;
```

Comment:
I would recommend from the start annotating your packages with `@ParametetersAreNonNullByDefault`

Reply:
Ok, will do in the next PR